rampwf/utils/cli/tests/test_cli.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
```

Comment:
@kegl why not building on existing tests here https://github.com/paris-saclay-cds/ramp-workflow/blob/master/rampwf/utils/tests/test_command_line.py#L435 ?

Reply:
Those test the old cli (`ramp_test_submission`). The new added by @glemaitre seemed easier for me to start with. It was untested, now it's covered. These are printing functions, no output, no other modules depend on this. I don't want to test either the content of the tables or the actual format. The current test protects against inadvertent modification of the printing routing.

We should perhaps deprecate the old cli. Two remarks:
1. The dispatcher still uses the old cli.
2. the command `ramp` is created only when ramp-board is installed, maybe ramp-workflow should also do it.